docs/tutorial/spark-tutorial/t-spark-submit.md

```diff
@@ -0,0 +1,36 @@
+The Charmed Spark solution offers several ways to interact with the Spark cluster. They are:
+1. Using Spark Submit
+2. Using Spark Shell
+
+In this section, we'll run a sample job using `spark-submit` CLI command provided with `spark-client` snap. The sample Python program that we are going to run have already been uploaded to S3 in the earlier section. Also, the sample data file has also been uploaded to `data/data.csv.gz` in the same S3 bucket. Run the Python script with `spark-submit` as follows:
+
+```bash
+spark-client.spark-submit \
+  --username spark --namespace spark \
+  --deploy-mode cluster \
+  s3a://$S3_BUCKET/scripts/stock_country_report.py \
+    --input  s3a://$S3_BUCKET/data/data.csv.gz \
+    --output s3a://$S3_BUCKET/data/output_report_microk8s
```

Comment:
Maybe it could be good, building on top of the spark shell, to show, describe and split the python script, otherwise it would seem a bit too much of dark magic